lang/java/reef-runtime-mock/src/main/java/org/apache/reef/mock/request/AllocateEvaluator.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.reef.mock.request;
+
+import org.apache.reef.driver.evaluator.FailedEvaluator;
+import org.apache.reef.mock.ProcessRequest;
+import org.apache.reef.mock.runtime.MockAllocatedEvalautor;
+import org.apache.reef.mock.runtime.MockFailedEvaluator;
+
+/**
+ * Allocate Evaluator process request.
+ */
+public final class AllocateEvaluator implements
```

Comment:
Is the spelling right? `Allocate` sounds like this is a verb, which is unusual for a class name.

Reply:
The verb usage was intentional in all Process Requests, indicating what's to be done. Is that okay?

Reply:
It does seem to make sense in this context.